### PR TITLE
Fixes the attractor example

### PR DESCRIPTION
**Pull request by jonathanrocher**
This PR removes 3D models from the model list of the 2D visualization task.

Fixes https://github.com/enthought/envisage/issues/52

### CODE REVIEW

**Comment by jonathanrocher** [comment]
@dpinte @corranwebster Mind taking a look? Trivial tiny PR... (and py3.4 failure is on master too)

File: examples/plugins/tasks/attractors/visualize_2d_task.py [hunk 1]
```diff
@@ -64,9 +65,7 @@ def _default_layout_default(self):
 
     def _models_default(self):
         from model.henon import Henon
-        from model.lorenz import Lorenz
-        from model.rossler import Rossler
-        return [ Henon(), Lorenz(), Rossler() ]
+        return [Henon()]
```
Comment (dpinte) [inline examples/plugins/tasks/attractors/visualize_2d_task.py]: I haven't looked at the details. Why are `Lorenz` and `Rossler` models removed? are they broken?

Comment (dpinte) [inline examples/plugins/tasks/attractors/visualize_2d_task.py]: ha... reading the description of the PR answers the question ... `Lorenz` and `Rossler` are 2D models.

Comment (dpinte) [inline examples/plugins/tasks/attractors/visualize_2d_task.py]: but both IModel3D have adapters `IModel3dIPlottable2dMixin`, so they should be compatible with a 2D view.

Comment (jonathanrocher) [inline examples/plugins/tasks/attractors/visualize_2d_task.py]: Yep, seems like I didn't understand the example correctly. Do you seem the same failure as me?

**Comment by mdickinson** [comment]
I've just opened an alternative fix in #103. @jonathanrocher: do you have a moment to test it?

**Comment by jonathanrocher** [comment]
Closing in favor of @mdickinson 's fix #103 .

**Pull request closed without merging**